describe.py

```diff
@@ -516,6 +534,7 @@ def generate_all_api_documents(
                 api["discoveryRestUrl"],
                 doc_destination_dir,
                 artifact_destination_dir,
+                discovery_uri_template,
```

Comment:
Consider having `document_api` not take an extra parameter:

```python
            document_api(
                api["name"],
                api["version"],
                discovery_uri_template or FLAGS.discovery_uri_template or api["discoveryRestUrl"],
                doc_destination_dir,
                artifact_destination_dir,              
            )
```

Then, in `document_api`, you can simply have
```python
uri = uri.expand(...)
```
without any conditionals.

Note also that I have `FLAGS.discovery_uri_template` in the second position; that makes more sense, I think. But I also think we shouldn't be referencing `FLAGS` directly from within the functions, but it should be passed in.


```

Reply:
Also, I'm not sure why the flag `--discovery_uri_template` still defaults to `DISCOVERY_URI`. I think we should change the latter symbol to the value of `DISCOVERY_URI_TEMPLATE` that you  have in scripts/updatediscoveryartifacts.py

Reply:
Fixed. Please could you take another look?